packages/manager-components/src/components/table/table.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropsWithChildren } from 'react';
+import './table.scss';
+
+export type TableProps = PropsWithChildren<{
+  fullWidth?: boolean;
+  cellSpacing?: number;
+}>;
+
+export const Table: React.FC<TableProps> = ({
+  children,
+  fullWidth = false,
+  cellSpacing = 0,
+}) => (
+  <table
+    className={`manager-table ${fullWidth ? 'w-full' : ''}`}
```

Comment:
i would recommend using css module to avoid className conflicts

Reply:
Sure, Good Idea 👍